### PR TITLE
Add figure captions

### PR DESCRIPTION
**Pull request by luizaandrade**

### CODE REVIEW

File: auxiliary/conclusion.tex [hunk 1]
```diff
@@ -60,6 +60,7 @@
 	\begin{figure}
 		\centering
 		\includegraphics[width=1.5\linewidth]{diagrams/Conclusion}
+		\caption{Research data work outputs}
```
Comment (mariaruth) [inline auxiliary/conclusion.tex:63]: what about, 'development research data outputs'? just to make this more aligned with the book title

File: chapters/0-introduction.tex [hunk 2]
```diff
@@ -153,6 +153,7 @@ \subsection*{The DIME Wiki: A complementary resource}
 	\begin{figure}
 		\centering
 		\includegraphics[width=1.5\linewidth]{diagrams/Introduction}
+		\caption{Overview of development research data work tasks}
```
Comment (mariaruth) [inline chapters/0-introduction.tex:156]: suggest dropping 'work'

File: chapters/4-acquisition.tex [hunk 3]
```diff
@@ -1003,6 +1003,10 @@ \section{Looking ahead}
 Finally, it emphasized the secure handling of this potentially sensitive information,
 giving you a range of tools and a complete workflow
 for transferring and storing data securely at all times.
+Figure 2 shows an overview of the tasks and outputs discussed. 
+Note that the data acquisition process often occurs repeated times, 
+and commonly takes both of the paths portrayed in the figure in different occasions, 
+particularly when multiple data sources are involved.
```
Comment (mariaruth) [inline chapters/4-acquisition.tex:1009]:
```suggestion
Note that projects using multiple data sources will 
repeat the process portrayed in the figure multiple times, 
and may take both of the paths portrayed in the figure on different occasions. 
```

File: chapters/5-processing.tex [hunk 4]
```diff
@@ -873,6 +873,7 @@ \section{Looking ahead}
 	\begin{figure}
 		\centering
 		\includegraphics[width=1.6\linewidth]{diagrams/Cleaning}
+		\caption{Data processing input, tasks, and outputs}
```
Comment (mariaruth) [inline chapters/5-processing.tex:876]:
```suggestion
		\caption{Data processing inputs, tasks, and outputs}
```

File: chapters/6-analysis.tex [hunk 5]
```diff
@@ -812,6 +812,7 @@ \section{Looking ahead}
 	\begin{figure}
 		\centering
 		\includegraphics[width=1.6\linewidth]{diagrams/Analysis}
+		\caption{Data analysis input, tasks, and outputs}
```
Comment (mariaruth) [inline chapters/6-analysis.tex:815]:
```suggestion
		\caption{Data analysis inputs, tasks, and outputs}
```

**Review by mariaruth — APPROVED** [review]
Thanks for catching these gaps, Luiza! A few minor changes to wording suggested.

**Comment by luizaandrade** [comment]
closing this PR as changes were made to the bookdown branch.

**Pull request closed without merging**